mobile/Screens/MindBodyScreen.jsx

```diff
@@ -1,26 +1,44 @@
-import React, { useState, useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import {
-  StyleSheet, Text, View, Button,
+  View, Animated, FlatList,
 } from 'react-native';
 import axios from 'axios';
+import MindBodyCard from '../Components/MindBodyCard';
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: '#fff',
-    alignItems: 'center',
-    justifyContent: 'center',
-  },
-});
+function Card() {
+  const [scrollViewWidth, setScrollViewWidth] = React.useState(0);
+  const [scrollViewHeight, setScrollViewHeight] = React.useState(0);
+  const boxWidth = scrollViewWidth * 0.8;
+  // const boxHeight = scrollViewHeight * 0.5;
+  const boxDistance = scrollViewWidth - boxWidth;
+  const halfBoxDistance = boxDistance / 2;
+  const pan = React.useRef(new Animated.ValueXY()).current;
+  const [data, setData] = useState([
```

Comment:
We might want to rename this to `mindBodyDeck` or something a little more specific